src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static Task[] taskList = new Task[100];
+    private static int counter = 0;
+    private static final String LINE = "___________________________________________";
+
+    public static void printGreeting() {
+        System.out.println("\t" + LINE);
+        System.out.println("\t Hello I'm Duke, your personal chatbot.");
+        System.out.println("\t Is there anything I can do for you");
+        System.out.println("\t" + LINE);
+    }
+    public static void printList() {
+        System.out.println("\t" + LINE);
+        System.out.println("\t Here are the tasks for your list: ");
+        for(int i=0; i< counter; i++) {
+            System.out.println("\t " + (i+1) + "." + taskList[i]  +  taskList[i].description);
+        }
+        System.out.println("\t" + LINE);
+    }
+
+    public static void printTask(Task task) {
+        System.out.println("\t" + LINE);
+        System.out.println("\t Got it. I've added this task:");
+        System.out.println("\t  " + task);
+        System.out.println("\t Now you have " + counter + " tasks in your list.");
+        System.out.println("\t" + LINE);
+    }
+
+    public static void printBye() {
+        System.out.println("\t" + LINE);
+        System.out.println("\t Bye! Do let me know if you need any further assistance");
+        System.out.println("\t" + LINE);
+    }
+    public static void markTaskAndPrint(int taskIndex) {
+        if(taskIndex < 0  || taskList[taskIndex] == null) {
+            System.out.println("\t" + LINE);
+            System.out.println("\t Please ensure that you enter the correct task number");
+            System.out.println("\t" + LINE);
+            return;
+        }
+        taskList[taskIndex].isDone = true;
```

Comment:
Would be better to handle cases when the user tries to mark an already marked task.